src/Data/Impact/Definition.elm

```diff
@@ -1,10 +1,11 @@
-module Data.Impact.Definition exposing (AggregatedScoreData, Definition, Definitions, Quality(..), Source, Trigram(..), decode, forScope, get, isAggregate, toString, toTrigram, trigrams)
+module Data.Impact.Definition exposing (AggregatedScoreData, Definition, Definitions, DefinitionsBase, Quality(..), Source, Trigram(..), decode, decodeBase, encodeBase, filter, foldl, forScope, get, init, isAggregate, map, toString, toTrigram, trigrams, update)
```

Comment:
Would you mind spreading these over multiple lines as we did for other modules? Thanks!